doc/rtd/applications.rst

```diff
@@ -1557,6 +1561,304 @@ Command Line Arguments
+    so a "24h" duration correspond to 5 time steps in 6-hourly data (i.e. 0h,6,12,18,24UTC).
+    (default: 10)
+
+--min_endpoint_distance MIN_ENDPOINT_DIST
+    The minimum great-circle distance between the first candidate on a path and the last candidate (in degrees).
+    (default: 0.0)
+
+--min_path_distance MIN_PATH_DISTANCE
+    The minimum accumulated great-circle distance between nodes in a path (in degrees).
+    (default: 0.0)
+
+--min_path_length MIN_PATH_LENGTH
+    Minimum path length (default: 0.0)
+
+--max_gap MAX_GAP
+    The number of allowed missing points between spatially proximal candidate nodes
+    while still considering them part of the same path.
+    (default: 3)
+
+--track_threshold_cmd TRACK_THRESHOLD_CMD
+    Filter paths based on the number of times where a particular threshold is satisfied.
+    Threshold commands are separated by a semicolon ("<cmd1>;<cmd2>;...").
+    Each threshold command takes the form "col,op,value,count".
+    These arguments are as follows.
+    col is the name of the column to use for thresholding, as specified in --in_fmt.
+    op is the operator that must be satisfied for threshold (options include >,>=,<,<=,=,!=,|>=,|<=).
+    value is the value on the right-hand-side of the comparison.
+    count is either the minimum number of time slices where the threshold must be satisfied
+    or the instruction "all", "first", or "last".
+    Here "all" is used to indicate the threshold must be satisfied at all points along the path,
+    "first" is used to indicate the threshold must be satisfied only at the first point along the path, and
+    "last" is used to indicate the threshold must be satisfied only at the last point along the path.
+    (default: surface_wind_speed,>=,10.0,10;lat,<=,50.0,10;lat,>=,-50.0,10;GEOPOTENTIAL_AT_SURFACE,<=,15.0,10)
+
+--prioritize VARIABLE_NAME
+    Variable to use when prioritizing path
+
+--allow_repeated_times true/false
+    Allow repeated times (default: false)
+
+--cal_type CAL_TYPE
+    Calendar type (default: standard)
+
+--first_step FIRST_STEP
+    first time step to process (default: 0)
+
+--last_step LAST_STEP
+    last time step to process (default: -1)
+
+--time_filter TIME_FILTER
+    stride to process time steps at (default: 1)
+
+--n_threads NUMBER_OF_THREADS
+    Sets the thread pool size on each MPI rank. When the default value of -1 is used TECA will
+    coordinate the thread pools across ranks such each thread is bound to a unique physical core.
+    (default: -1)
+
+--help
+    displays documentation for application specific command line options
+
+--advanced_help
+    displays documentation for algorithm specific command line options
+
+--full_help
+    displays both basic and advanced documentation together
+
+--verbose VERBOSE
+    Enable verbose output (default: 0)
+
+Examples
+~~~~~~~~
+
+ERA5 data
+^^^^^^^^^
+
+.. code-block:: bash
+
+    #!/bin/bash
+    #SBATCH -C cpu
```

Comment:
can I suggest that this example be modified to (1) demonstrate running on the GPU, and (2) to replicate exactly the 2017 run that you did that we verified against Paul and Colins's TempestExtremes runs?

Reply:
This is the exactly 2017 run that I did. What is important here is how you will define the input file (era5_2017.mcf).
I can demonstrate using the GPU, but it is much slower than the CPU.